tests/resources/test_descriptors.py

```diff
@@ -50,6 +50,21 @@ def test_from_predictor_responses():
         .format(descriptors.project_id)
     assert session.last_call.method == 'POST'
 
+    graph = GraphPredictor(
+        name="Graph",
+        description="Contains a featurizer",
+        predictors=[featurizer]
+    )
+    graph_results = descriptors.from_predictor_responses(predictor=graph, inputs=[MolecularStructureDescriptor(col)])
+    assert graph_results == [
+        RealDescriptor(
+            key=r['descriptor_key'],
+            lower_bound=r['lower_bound'],
+            upper_bound=r['upper_bound'],
+            units=r['units']
+        ) for r in response_json['responses']
+    ]
+
```

Comment:
What new logic is this testing?

Reply:
The fact that the `from_predictor_responses` endpoint accepts a `Union[PredictorNode, GraphPredictor]` type, and has a conditional checking how to extract the JSON payload.